src/main/resources/db/migration/V84__barn_paa_behandling.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE behandling_barn (
+    id                UUID PRIMARY KEY,
+    behandling_id     UUID REFERENCES behandling (id),
```

Comment:
(+ formattering :P )
```suggestion
    behandling_id  NOT NULL   UUID REFERENCES behandling (id),
```